src/invoices/InvoicesList/InvoiceLine/InvoiceLineDetailsLayer.js

```diff
@@ -1,103 +1,146 @@
-import React, { Component, Fragment } from 'react';
+import React, { Fragment, useCallback, useEffect, useState } from 'react';
 import PropTypes from 'prop-types';
-import { get } from 'lodash';
 import ReactRouterPropTypes from 'react-router-prop-types';
 
 import {
+  baseManifest,
   LoadingPane,
   Tags,
+  useModalToggle,
 } from '@folio/stripes-acq-components';
 
 import {
   invoiceLineResource,
   invoiceResource,
 } from '../../../common/resources';
+import { PO_LINES_API } from '../../../common/constants';
 import InvoiceLineDetails from '../../InvoiceLineDetails';
 
-class InvoiceLineDetailsLayer extends Component {
-  static manifest = Object.freeze({
-    invoiceLine: invoiceLineResource,
-    invoice: invoiceResource,
-    query: {},
-  });
-
-  static propTypes = {
-    match: ReactRouterPropTypes.match,
-    mutator: PropTypes.object.isRequired,
-    resources: PropTypes.object.isRequired,
-    showToast: PropTypes.func.isRequired,
-  }
-
-  state = {
-    isTagsPaneOpened: false,
+const InvoiceLineDetailsLayer = ({
+  history,
+  match: { params },
+  mutator,
+  showToast,
+}) => {
+  const [polNumber, setPolNumber] = useState('');
+  const [invoice, setInvoice] = useState({});
+  const [invoiceLine, setInvoiceLine] = useState({});
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
it might be `true` as default value, however, you should set it to true manually at the beginning of loading function, in case you call this after component is created several times.